### PR TITLE
Add support for Cocoapods

### PR DESCRIPTION
**Pull request by dlowder-salesforce**

### CODE REVIEW

File: ReactNativeUiLib.podspec [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  s.name         = "ReactNativeUiLib"
+  s.version      = package['version']
+  s.summary      = "React Native UI Library"
+
+  s.authors      = "Wix.com"
+  s.homepage     = "https://github.com/wix/react-native-ui-lib"
+  s.license      = package['license']
+  s.platforms    = { :ios => "9.0", :tvos => "9.2" }
+
+  s.module_name  = 'ReactNativeNavigation'
```
Comment (Inbal-Tish) [inline ReactNativeUiLib.podspec]: module_name is wrong

File: ReactNativeUiLib.podspec [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  s.source       = { :git => "https://github.com/wix/react-native-ui-lib.git", :tag => "#{s.version}" }
+  s.source_files  = "lib/ios/**/*.{h,m}"
```
Comment (Inbal-Tish) [inline ReactNativeUiLib.podspec]: Path is wrong

File: ReactNativeUiLib.podspec [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = "ReactNativeUiLib"
+  s.version      = package['version']
+  s.summary      = "React Native UI Library"
+
+  s.authors      = "Wix.com"
+  s.homepage     = "https://github.com/wix/react-native-ui-lib"
```
Comment (Inbal-Tish) [inline ReactNativeUiLib.podspec]: Better to add the 'homepage' to the package.json and read it from there

File: ReactNativeUiLib.podspec (hunk 2 above)
Comment (dlowder-salesforce) [reply ReactNativeUiLib.podspec]: Actualy it is correct.  The top-level `ios/` path is for a demo app.

File: ReactNativeUiLib.podspec (hunk 3 above)
Comment (dlowder-salesforce) [reply ReactNativeUiLib.podspec]: Done.

File: ReactNativeUiLib.podspec (hunk 1 above)
Comment (dlowder-salesforce) [reply ReactNativeUiLib.podspec]: Fixed.

**Comment by aathapa** [comment]
this is much needed since 0.60

**Pull request merged**